src/components/Chart.jsx

```diff
@@ -93,6 +93,7 @@ class Chart extends Component {
                             <div className="ciq-chart-area">
                                 <div className="ciq-chart">
                                     <RenderInsideChart at="holder">
+                                        {children}
```

Comment:
@shayanbinary Regarding the description in the chartIQ site, it'd be better to place the markers in the `subholder`.
![image](https://user-images.githubusercontent.com/16184705/54971624-474d2a80-4fc2-11e9-9bb7-3b679c7980e6.png)